### PR TITLE
docs: Document supported BiDi entities

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Related to https://github.com/appium/appium-xcuitest-driver/pull/2458/

### CODE REVIEW

File: docs/reference/bidi.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+# Supported Events
+
+## log.entryAdded
+
+This event is emitted if the driver retrieves a new entry for any of the below log types. Logs collection might be disabled by the `appium:skipLogCapture` capability.
+
+### crashlog
+
+Events are emitted for both emulator and real devices. The latter only works if [py-ios-device](hhttps://github.com/YueChen-C/py-ios-device) is installed on the server host. Each event contains a particular device crash report entry.
```
Comment (eglitise) [inline docs/reference/bidi.md:17]: Typo in link (`hhttps`)

File: docs/reference/bidi.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Events are always emitted with the `NATIVE_APP` context.
```
Comment (eglitise) [inline docs/reference/bidi.md:18]: Not really an issue, but lines separated by a single newline will be shown on the same line in MkDocs. New paragraphs require two newlines. The current format might be a bit difficult to read with the sentences all starting with 'Events'.
I think using bulleted lists would make it cleaner.

File: docs/reference/bidi.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+### syslog
+
+Events are emitted for both emulator and real devices. Each event contains a single device system log line.
+Events are always emitted with the `NATIVE_APP` context.
+
+### safariConsole
+
+Events are emitted for both emulator and real devices. Each event contains a single safari console log line.
```
Comment (eglitise) [inline docs/reference/bidi.md:27]: I guess Safari should be capitalised (same below)

**Review by eglitise — COMMENTED** [review]
👍

File: docs/reference/bidi.md (hunk 1 above)
Comment (mykola-mokhnach) [reply docs/reference/bidi.md:17]: good catch, fixed

File: docs/reference/bidi.md (hunk 2 above)
Comment (mykola-mokhnach) [reply docs/reference/bidi.md:18]: It's ok if it is in the same paragraph. I just prefer to make .md files in such style

File: docs/reference/bidi.md (hunk 3 above)
Comment (mykola-mokhnach) [reply docs/reference/bidi.md:27]: fixed

**Review by eglitise — APPROVED** [review]

**Pull request merged**